eyieko/landville-frontend
# #167994901 Set Page Meta data for Existing Pages and Add SSR

## Description ##
- Search engines are not great at indexing Javascript bundles, This PR adds some tweaks to render the pages HTML using nest js and Angular Universal.
- This way social media and search engine bots can easily index the HTML which improves SE0.
-  Shifting rendering to the server also increases the first-page download speed.

## Type of change ##
Please select the relevant option

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes

## Checklist: ##

## How to test this PR
- View Page source of each page and you will be able to see HTML of the page and also metadata details like the title and Page descriptions

## Related Pivotal Tracker stories ##
[#167994901](https://www.pivotaltracker.com/story/show/167994901)
